src/test/java/de/tum/in/www1/artemis/theia/TheiaInfoContributorTest.java

```diff
@@ -0,0 +1,36 @@
+package de.tum.in.www1.artemis.theia;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_THEIA;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.info.Info;
+import org.springframework.context.annotation.Profile;
+
+import de.tum.in.www1.artemis.config.Constants;
+import de.tum.in.www1.artemis.service.theia.TheiaInfoContributor;
+
+@Profile(PROFILE_THEIA)
+class TheiaInfoContributorTest {
+
+    @Value("${theia.portal-url}")
+    private String expectedValue;
+
+    TheiaInfoContributor theiaInfoContributor;
+
+    @Test
+    void testContribute() {
+        Info.Builder builder = new Info.Builder();
+        theiaInfoContributor = new TheiaInfoContributor();
+        try {
+            theiaInfoContributor.contribute(builder);
+        }
+        catch (NullPointerException e) {
```

Comment:
Should the test really pass when there is a NullPointer? If so, maybe add a comment why this exception is ok

Reply:
Fair point, I added that initially to stop missing configuration files - but that doesn't really make sense obv. Fixed it